compiler/x/codegen/OMRMachine.cpp

```diff
@@ -279,7 +279,10 @@ OMR::X86::Machine::findBestFreeGPRegister(TR::Instruction   *currentInstruction,
          first = TR::RealRegister::FirstMMXR;
          last  = TR::RealRegister::LastMMXR;
          break;
+      case TR_VectorReg256:
+      case TR_VectorReg512:
       case TR_QuadWordReg:
```

Comment:
I believe this switch statements should contain a case for TR_VectorReg128

Reply:
Agree.